mkosi.md

```diff
@@ -335,6 +335,14 @@ a boolean argument: either "1", "yes", or "true" to enable, or "0",
   Linux, additional repositories must be passed in the form `<name>::<url>`
   (e.g. `myrepo::https://myrepo.net`).
 
+`UseSystemRepositories=`, `--use-system-repositories`
+
+: Allows use of the system's existing software package repositories.
+  By default a hardcoded set of default repositories for each distribution
```

Comment:
Currently this is an dnf-only thing, right? If so, it would be good to note that here.

Reply:
Yes, good point.

I've taken a stab at updating this to identify that it is dnf-only.